docs/troubleshooting-guide/sync-fingerprint-scan.md

```diff
@@ -0,0 +1,82 @@
+
+**How to detect this:**
+
+1. The UI isn't displaying some of the values you expect to see in a dropdown menu.
+2. The UI is showing a search box for selecting values where you expect a dropdown menu.
+
+**How to fix this:**
+
+1. Go to the Admin Panel and select "Data Model"
+2. Select the database, schema, table and field in question.
+3. Click the gear-icon to view all the field's settings.
+4. Set "Field Type" to "Category" and "Filtering on this field" to "A list of all values"
+5. Click the button "Re-scan this field" in the bottom.
+
+### I cannot force Metabase to sync or scan using the API
+
+Metabase syncs and scans regularly, but if the database administrator has just changed the database schema, or if a lot of data is added automatically at specific times, you may want to write a script that uses the [Metabase API][metabase-api] to force sync or scan to take place right away. There are two ways to do this:
+
+1. Using an endpoint with a session token: `/api/database/:id/sync_schema` or `api/database/:id/rescan_values`. These do the same things as going to the database in the Admin Panel and choosing "Sync database schema now" or "Re-scan field values now" respectively. In this case you have to authenticate with a user ID and pass a session token in the header of your request.
+
+2. Using an endpoint with an API key: `/api/notify/db/:id`. This endpoint was made to notify Metabase to sync after an [ETL operation][etl] finishes. In this case you must pass an API key by defining the `MB_API_KEY` environment variable.
+
+**How to detect this:** Your script fails to run.
+
+**How to fix this:**
+
+1. Make sure you are able to sync and scan manually via the Admin Panel.
+2. Check the URL you are using to send the request to Metabase.
+3. Check the error message returned from Metabase.
+4. Check the credentials you are using to authenticate and make sure they identify your script as a user with administrative privileges.
+
+### Sync and scan take a very long time to run
+
+**How to detect this:** Sync and scan take a long time to complete.
+
+**How to fix this:** 
+1. For sync, this is usually caused by a large database with hundreds of schema, thousands of table and with hundreds of columns in each table. If only few of those are needed in Metabase, then restricting the privileges used to connect to the database with will make sure that Metabase can only sync a limited subset of the database.
+2. Scanning normally takes longer than sync, but changing the amount of fields which has "Filtering on this field" set to "A list of all values" to either "Search box" or "Plain input box" will mean that those will not be included in the scan.
+
+You can "fix" this by disabling scan entirely by going to the database in the Admin Panel and telling Metabase, "This is a large database," and then going to the Scheduling tab. However, sync is necessary: without it, Metabase won't know what tables exist or what columns they contain.
```

Comment:
Do we even want to recommend disabling scans?

Reply:
Johannes wanted it in with the disclaimer.